src/App.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color: #FFF;
   min-height: 100vh;
```

Comment:
Is the logo gonna be really big? Right now it is almost the size of my screen. I suggest setting a max-height property

Reply:
oh i didnt change any of the properties, this is just me putting the logo in just to have it in for now